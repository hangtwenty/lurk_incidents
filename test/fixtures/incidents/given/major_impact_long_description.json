{
    "backfilled": false,
    "components": [
        {
            "created_at": "2014-04-03T19:36:07.189Z",
            "description": null,
            "group_id": "vddt835f9d28",
            "id": "809yb3s4yt0f",
            "name": "Email",
            "page_id": "y2j98763l56x",
            "position": 1,
            "status": "operational",
            "updated_at": "2016-09-13T18:35:54.699Z"
        },
        {
            "created_at": "2013-02-16T15:30:09.678Z",
            "description": null,
            "group_id": null,
            "id": "t4gdf1wndjn3",
            "name": "Hosted Pages",
            "page_id": "y2j98763l56x",
            "position": 1,
            "status": "operational",
            "updated_at": "2016-08-25T05:50:34.000Z"
        },
        {
            "created_at": "2013-06-24T19:02:46.151Z",
            "description": null,
            "group_id": "744598q7pc53",
            "id": "w28xrs593c1v",
            "name": "Pingdom",
            "page_id": "y2j98763l56x",
            "position": 1,
            "status": "operational",
            "updated_at": "2016-09-14T08:45:06.567Z"
        },
        {
            "created_at": "2013-11-19T02:42:37.132Z",
            "description": null,
            "group_id": null,
            "id": "mwtxk4yc4l1p",
            "name": "API",
            "page_id": "y2j98763l56x",
            "position": 2,
            "status": "operational",
            "updated_at": "2016-08-12T19:48:21.000Z"
        },
        {
            "created_at": "2015-03-01T05:53:33.168Z",
            "description": null,
            "group_id": "744598q7pc53",
            "id": "sd12wq20r9zg",
            "name": "New Relic",
            "page_id": "y2j98763l56x",
            "position": 2,
            "status": "operational",
            "updated_at": "2016-02-18T22:49:34.841Z"
        },
        {
            "created_at": "2015-03-01T05:53:02.278Z",
            "description": null,
            "group_id": "744598q7pc53",
            "id": "80d9xq65zjxc",
            "name": "Librato",
            "page_id": "y2j98763l56x",
            "position": 3,
            "status": "operational",
            "updated_at": "2016-02-18T22:49:35.110Z"
        },
        {
            "created_at": "2013-02-16T15:29:50.682Z",
            "description": null,
            "group_id": null,
            "id": "bp20v9n8xv07",
            "name": "Management Interface",
            "page_id": "y2j98763l56x",
            "position": 3,
            "status": "operational",
            "updated_at": "2016-09-14T07:41:03.000Z"
        },
        {
            "created_at": "2015-02-24T21:47:31.155Z",
            "description": null,
            "group_id": "vddt835f9d28",
            "id": "5hk40m6bwys9",
            "name": "Webhook",
            "page_id": "y2j98763l56x",
            "position": 3,
            "status": "operational",
            "updated_at": "2016-02-18T22:49:32.781Z"
        },
        {
            "created_at": "2013-02-16T15:29:38.814Z",
            "description": null,
            "group_id": null,
            "id": "52y4k3q1hc8s",
            "name": "Website",
            "page_id": "y2j98763l56x",
            "position": 4,
            "status": "operational",
            "updated_at": "2016-08-25T14:21:24.929Z"
        },
        {
            "created_at": "2015-03-01T05:53:19.818Z",
            "description": null,
            "group_id": "744598q7pc53",
            "id": "gjb216lq5bv3",
            "name": "Datadog",
            "page_id": "y2j98763l56x",
            "position": 4,
            "status": "operational",
            "updated_at": "2016-02-18T22:49:35.694Z"
        },
        {
            "created_at": "2015-03-31T15:28:23.871Z",
            "description": null,
            "group_id": "vddt835f9d28",
            "id": "31x6gx047j70",
            "name": "Twitter",
            "page_id": "y2j98763l56x",
            "position": 4,
            "status": "operational",
            "updated_at": "2016-06-03T00:57:18.908Z"
        },
        {
            "created_at": "2013-03-11T19:16:38.903Z",
            "description": null,
            "group_id": null,
            "id": "63rdzfjtljky",
            "name": "Third Party Components",
            "page_id": "y2j98763l56x",
            "position": 5,
            "status": "operational",
            "updated_at": "2016-02-18T22:49:25.977Z"
        },
        {
            "created_at": "2014-04-09T11:58:58.225Z",
            "description": "Alerts coming from Pingdom, New Relic, and Webmon",
            "group_id": null,
            "id": "4c9qnv55m863",
            "name": "Component Alert Parsing",
            "page_id": "y2j98763l56x",
            "position": 6,
            "status": "operational",
            "updated_at": "2016-03-22T22:35:07.696Z"
        },
        {
            "created_at": "2014-05-30T22:21:18.474Z",
            "description": null,
            "group_id": null,
            "id": "znlwm435tzfk",
            "name": "Chat Integrations",
            "page_id": "y2j98763l56x",
            "position": 7,
            "status": "operational",
            "updated_at": "2016-06-10T22:59:14.696Z"
        }
    ],
    "created_at": "2016-02-18T14:24:07.383-08:00",
    "id": "lphfkdlhdbbd",
    "impact": "critical",
    "impact_override": "critical",
    "incident_updates": [
        {
            "affected_components": null,
            "body": "This incident has been resolved.",
            "created_at": "2016-02-18T16:36:47.081-08:00",
            "display_at": "2016-02-18T16:36:47.081-08:00",
            "id": "lmbkmjkkdhml",
            "incident_id": "lphfkdlhdbbd",
            "status": "resolved",
            "twitter_updated_at": "2016-02-19T00:36:48.785Z",
            "updated_at": "2016-02-18T16:36:48.785-08:00",
            "wants_twitter_update": true
        },
        {
            "affected_components": null,
            "body": "After a deploy, upgrading a library, our web application failed to start. We pushed a fix and brought back our services. We will continue to monitor the service's health, but our services are all operational. Our public metrics are delayed as a result of the downtime.",
            "created_at": "2016-02-18T14:44:47.857-08:00",
            "display_at": "2016-02-18T14:44:47.857-08:00",
            "id": "mcfjldlnfffn",
            "incident_id": "lphfkdlhdbbd",
            "status": "monitoring",
            "twitter_updated_at": "2016-02-18T22:44:48.777Z",
            "updated_at": "2016-02-18T14:44:48.777-08:00",
            "wants_twitter_update": true
        },
        {
            "affected_components": null,
            "body": "We had issues with some parts of our infrastructure starting after a deploy. We're working on a fix right now.",
            "created_at": "2016-02-18T14:24:07.541-08:00",
            "display_at": "2016-02-18T14:24:07.541-08:00",
            "id": "nghlhlmhgpdl",
            "incident_id": "lphfkdlhdbbd",
            "status": "identified",
            "twitter_updated_at": "2016-02-18T22:24:09.261Z",
            "updated_at": "2016-02-18T14:24:09.261-08:00",
            "wants_twitter_update": true
        }
    ],
    "monitoring_at": "2016-02-18T14:44:47.857-08:00",
    "name": "Errant deploy causing site outage",
    "page_id": "y2j98763l56x",
    "postmortem_body": null,
    "postmortem_body_last_updated_at": null,
    "postmortem_ignored": false,
    "postmortem_notified_subscribers": false,
    "postmortem_notified_twitter": false,
    "postmortem_published_at": null,
    "resolved_at": "2016-02-18T16:36:47.081-08:00",
    "scheduled_auto_completed": false,
    "scheduled_auto_in_progress": false,
    "scheduled_for": null,
    "scheduled_remind_prior": false,
    "scheduled_reminded_at": null,
    "scheduled_until": null,
    "shortlink": "http://mstspg.co/Q2d",
    "status": "resolved",
    "updated_at": "2016-02-18T16:36:49.861-08:00"
}
