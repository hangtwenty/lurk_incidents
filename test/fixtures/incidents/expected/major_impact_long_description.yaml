narrative: >
    Errant deploy causing site outage

    We had issues with some parts of our infrastructure starting after a deploy. We're working on a fix right now.

    After a deploy, upgrading a library, our web application failed to start. We pushed a fix and brought back our services. We will continue to monitor the service's health, but our services are all operational. Our public metrics are delayed as a result of the downtime.

    This incident has been resolved.

    This incident affected: Notifications (Email, Webhook, Twitter), Hosted Pages, API, Management Interface, Website, Third Party Components, Component Alert Parsing, Chat Integrations, and Public Metrics (Pingdom, New Relic, Librato, Datadog).
